src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -64,25 +64,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return result;
     }
 
-    // Helper static function to ensure that all ambiguous-width glyphs are reported as narrow.
-    // See microsoft/terminal#2066 for more info.
-    static bool _IsGlyphWideForceNarrowFallback(const std::wstring_view /* glyph */)
-    {
-        return false; // glyph is not wide.
-    }
-
-    static bool _EnsureStaticInitialization()
-    {
-        // use C++11 magic statics to make sure we only do this once.
-        static auto initialized = []() {
-            // *** THIS IS A SINGLETON ***
-            SetGlyphWidthFallback(_IsGlyphWideForceNarrowFallback);
-
-            return true;
-        }();
-        return initialized;
-    }

```

Comment:
`CodepointWidthDetector` defaults to narrow glyphs if no fallback is specified already, so there's no need to do this.